src/test/java/com/ite/itea/ecommerce/adapters/out/persistence/product/JpaProductRepositoryImplTest.java

```diff
@@ -62,21 +64,17 @@ private void assertTestProduct(Product product) {
 
         final var customProduct = (CustomProduct) product;
 
-        assertThat(customProduct.id()).isEqualTo(new ProductId("3"));
-        assertThat(customProduct.name()).isEqualTo("Product Name");
-        assertThat(customProduct.description()).isEqualTo("Product Description");
-        assertThat(customProduct.imageName()).isEqualTo("Product Image Name");
-
-        assertThat(customProduct.getProductParts()).hasSize(2);
-        final var part1 = customProduct.getProductParts().get(0);
-        assertThat(part1.name()).isEqualTo("Part1 Name");
-        assertThat(part1.price()).isEqualTo(EuroPrice.ofCents(250));
-        assertThat(part1.quantity()).isEqualTo(2);
-
-        final var part2 = customProduct.getProductParts().get(1);
-        assertThat(part2.name()).isEqualTo("Part2 Name");
-        assertThat(part2.price()).isEqualTo(EuroPrice.ofCents(500));
-        assertThat(part2.quantity()).isEqualTo(3);
+        assertAll("Product",
+                () -> assertThat(customProduct.id()).isEqualTo(new ProductId("3")),
+                () -> assertThat(customProduct.name()).isEqualTo("Product Name"),
+                () -> assertThat(customProduct.description()).isEqualTo("Product Description"),
+                () -> assertThat(customProduct.imageName()).isEqualTo("Product Image Name"),
+                () -> assertThat(customProduct.getProductParts()).containsExactlyInAnyOrder(
```

Comment:
Nice, I didn't know about `containsExactlyInAnyOrder`. This makes it even cleaner in my opinion. 👌